talk/morelanguage/random.tex

```diff
@@ -0,0 +1,122 @@
+%\subsection[rnd]{Random}
```

Comment:
@sponce I am fine with merging the random slides into the previous `morestl` section, but I would like to keep the slides in a separate tex file. Should I remove the comment here as well? It makes it easier to add it back later, and also serves as a reminder that we did not "just forget to add a subsection".